release-team/role-handbooks/emeritus-adviser/README.md

```diff
@@ -61,7 +62,9 @@ Occasionally, a role may not get enough qualified candidates, in which case it's
 
 Weeks: 1 to 12
 
-First, have a "shadow orientation" meeting sometime in week 2 or week 3, after all shadows are selected.  This meeting will likely have to be split for time zone reasons and held twice.  In this meeting, go over the information in the main Shadow docs, give information about the release team and SIG Release, and then open things for questions or discussion. Make sure that Shadows know they can come to you if they have problems during their apprenticeship.
+Remove prior release shadows from the [kubernetes-release-team-shadows](https://groups.google.com/forum/#!forum/kubernetes-release-team-shadows) Google Group and add the current release shadows.  This list can facilitate easier communication between what at times is 2-3 dozen shadows.
```

Comment:
```suggestion
Remove prior release shadows from the [kubernetes-release-team-shadows](https://groups.google.com/forum/#!forum/kubernetes-release-team-shadows) Google Group and add the current release shadows. This list can facilitate easier communication between what at times is 2-3 dozen shadows.
```

Reply:
Add line breaks for easier future reviews.

Reply:
I'd meant to not add line breaks since the file didn't have them and that seemed to be the prevailing style.  If there's a strong preference on things like this can they be put in a linter?

Reply:
Ahhh. Missed that! Definitely prefer the prevailing style.
Eventually yes, we should lint all the things!

Reply:
Sohweet I think two spaces is prevailing too across the history of computer/typed text ;)